server/images.go

```diff
@@ -687,43 +687,18 @@ func CopyModel(src, dst model.Name) error {
 	return err
 }
 
-func deleteUnusedLayers(skipModelPath *ModelPath, deleteMap map[string]struct{}) error {
-	fp, err := GetManifestPath()
+func deleteUnusedLayers(deleteMap map[string]struct{}) error {
+	manifests, err := Manifests()
```

Comment:
This is a nice code reduction to use the Manifests function. However, did you want to only skip errors for invalid JSON? This will also ignore errors like file permission denied, etc.

Reply:
You're right. I missed that